### ISSUE TITLE
Configurer l'environnement dans les erreurs Sentry front

### ISSUE THREAD

**Issue by maudetes** [datagouv/data.gouv.fr#1438]
L'ajout de la variable environnement (dev.data.gouv.fr, etc.) a été fait sur le backend, mais pas sur le front (Vue 3 côté udata-front et Vue 1 côté admin).

Un projet distinct peut être créé côté Sentry pour séparer les issues front du back.

- [ ] [Exemple d'erreur](https://errors.data.gouv.fr/organizations/sentry/issues/140574/?project=12&query=is%3Aunresolved&referrer=issue-stream) côté admin udata : tenter une [recherche sur l'admin](https://www.data.gouv.fr/fr/admin/search?q=test).

- [x] [Exemple d'erreur](https://errors.data.gouv.fr/organizations/sentry/issues/144924/?project=12&query=&referrer=issue-stream&statsPeriod=14d) côté udata-front : se rendre sur https://www.data.gouv.fr/fr/datasets/559390f9c751df0139a453ba/ (soucis d'owner manquant sur les ressources communautaires)

**Comment by nicolaskempf57** [comment]
@maudetes @bolinocroustibat Est-ce qu'on ferme le ticket ? De la description, il semble qu'il reste juste l'admin à configurer mais mercredi on a dit qu'on ne le faisait pas

**Comment by bolinocroustibat** [comment]
Côté back c'est fait, côté front ce serait bien d'avoir un projet Sentry séparé, qui envoie l'environnement (dev, demo, prod) dans la variable adéquate.
Si pas possible pour l'instant on ferme le ticket

**Comment by nicolaskempf57** [comment]
On peut faire un projet séparé pour le JS mais pour le plugin python udata_front ça me parait compliqué. Est-ce qu'on veut qu'udata_front soit sur 2 projets différents en fonction du coté back / front ?

**Comment by maudetes** [comment]
Yes perso, vu la remarque de Nicolas, plutôt partante pour garder un unique projet Sentry tant qu'on a les views jinja udata-front !
Je ne vois plus d'erreur loggée par contre sur https://www.data.gouv.fr/fr/datasets/559390f9c751df0139a453ba/, right?

**Comment by nicolaskempf57** [comment]
@maudetes est-ce que ce ticket est toujours existant ?

**Comment by maudetes** [comment]
Nope, on peut clore!
